BinaryObjectScanner/Protection/InterLok.cs

```diff
@@ -23,10 +24,14 @@ public class InterLok : IExecutableCheck<PortableExecutable>
             return null;
         }
         
-        private static string GetVersion(string match)
+        private static string GetVersion(string matchMe)
         {
-            match = match.Remove(match.IndexOf(",")).Trim();
-            return match.Substring("InterLok ".Length + match.IndexOf("InterLok"));
+            var match = new Regex(@"(?<=InterLok )(.*?)(?=,)").Match(matchMe);
```

Comment:
Suggestion: Use `Regex.Match(input, pattern)` instead of creating a new Regex object.

Reply:
Done.